### PR TITLE
Create commons-beanutils.travis.yml

### PR DESCRIPTION
**Pull request by genisysram**

### CODE REVIEW

File: travis-ymls/commons-beanutils.travis.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+language: java
+sudo: false
+
+arch: 
+    - amd64
+    - ppc64le
+
+jdk:
+  - oraclejdk11
```
Comment (gururajrkatti) [inline travis-ymls/commons-beanutils.travis.yml:38]: This is not supported for ppc64le. Can we exclude this

File: travis-ymls/commons-beanutils.travis.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - openjdk8
+  - openjdk11
+  - openjdk15
+  - openjdk-ea
+
+before_install:
+        - mkdir -p /opt/maven
```
Comment (gururajrkatti) [inline travis-ymls/commons-beanutils.travis.yml:45]: Can we add as to why this manual installation of mvn required ? Also is it required for intel also ?

**Comment by genisysram** [comment]
@gururajrkatti 
Modified suggested changes

File: travis-ymls/commons-beanutils.travis.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+language: java
+sudo: false
+
+arch: 
+    - amd64
+    - ppc64le
+
+jdk:
+ # - oraclejdk11
```
Comment (gururajrkatti) [inline travis-ymls/commons-beanutils.travis.yml:37]: This needs to be excluded only for ppc64le

Comment (genisysram) [reply travis-ymls/commons-beanutils.travis.yml]: modified the suggested change

**Review by gururajrkatti — APPROVED** [review]
ok

**Pull request merged**